src/network/networkd-address.c

```diff
@@ -571,6 +571,21 @@ int address_configure(
 
         address->flags |= IFA_F_PERMANENT;
 
+        if (address->home)
+                address->flags |= IFA_F_HOMEADDRESS;
+
+        if (address->nodad)
+                address->flags |= IFA_F_NODAD;
+
+        if (address->mngtmpaddr)
+                address->flags |= IFA_F_MANAGETEMPADDR;
+
+        if (address->noprefixroute)
+                address->flags |= IFA_F_NOPREFIXROUTE;
+
+        if (address->autojoin)
+                address->flags |= IFA_F_MCAUTOJOIN;
```

Comment:
`IFA_F_MCAUTOJOIN` seems to be not defined in the project. The semaphore CI shows the following error message:

```bash
../src/network/networkd-address.c:587:35: error: ‘IFA_F_MCAUTOJOIN’ undeclared (first use in this function)
                 address->flags |= IFA_F_MCAUTOJOIN;
```

@martinpitt The Ubuntu build for s390 shows no error message, see [build log](https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-xenial-pitti-systemd-semaphore/xenial/s390x/s/systemd-upstream/20160922_081353@/log.gz). Is that CI systemd maybe testing the wrong version (`IFA_F_MCAUTOJOIN` is not defined and the build must fail in my opinion)?